css-shapes-2/Overview.bs

```diff
@@ -209,42 +236,55 @@ The ''shape()'' Function</h4>
 			Note: This starts a new subpath,
 			for the purpose of the ''close'' command.
 
-		<dt><dfn><<line-command>></dfn> = <dfn value>line</dfn> <<by-to>> <<coordinate-pair>>
+		<dt><dfn><<line-command>></dfn> = <dfn value>line</dfn> <<command-end-point>>
 		<dd>
 			Adds a <a href="https://www.w3.org/TR/SVG/paths.html#PathDataLinetoCommands">lineto</a> command
 			to the list of path data commands,
 			with an ending point specified by the <<coordinate-pair>>.
 
 			This draws a straight line from the command's starting point to its ending point.
 
-		<dt><dfn><<hv-line-command>></dfn> = [<dfn value>hline</dfn> | <dfn value>vline</dfn>] <<by-to>> <<length-percentage>>
+		<dt><dfn><<horizontal-line-command>></dfn> = hline [ to [ <length-percentage> | left | center | right | x-start | x-end ]
+															| by <length-percentage> ]
+		<dd>
+			Adds a horizontal
+			<a href="https://www.w3.org/TR/SVG/paths.html#PathDataLinetoCommands">lineto</a> command
+			to the list of path data commands.
+
+			This is equivalent to a ''line'' command
+			with the <<length-percentage>> given as the horizontal component of the <<coordinate-pair>>.
+			Specifying the horizontal component of <<position>> instead of a <<length-percentage>> (''left'', ''center'', ''right'', ''x-start'', or ''x-end''),
+			would draw a line to that <<position>>, with the <<position>>'s vertical component remaining the same as the starting point.
+
+		<dt><dfn><<vertical-line-command>></dfn> = vline [ to [ <length-percentage> | top | center | bottom | y-start | y-end ] | by <length-percentage> ]
 		<dd>
-			Adds a horizontal (for ''hline'') or vertical (for ''vline'')
+			Adds a vertical
 			<a href="https://www.w3.org/TR/SVG/paths.html#PathDataLinetoCommands">lineto</a> command
 			to the list of path data commands.
 
 			This is equivalent to a ''line'' command
-			with the <<length-percentage>> given as the horizontal or vertical component, respectively,
-			of the <<coordinate-pair>>,
-			and the other component specified appropriately to make the line horizontal or vertical.
+			with the <<length-percentage>> given as the vertical component of the <<coordinate-pair>>.
+			Specifying the horizontal component of <<position>> instead of a <<length-percentage>> (''top'', ''center'', ''bottom'', ''y-start'', or ''y-end''),
```

Comment:
```suggestion
			Specifying the horizontal component of <<position>>
			(''top'', ''center'', ''bottom'', ''y-start'', or ''y-end'')
			instead of a <<length-percentage>>,
```